build.sbt

```diff
@@ -404,6 +406,24 @@ lazy val client = libraryCrossProject("client")
   )
   .dependsOn(core, server % Test, testing % "test->test", theDsl % "test->compile")
 
+lazy val clientTestkit = libraryCrossProject("client-testkit")
+  .settings(
+    description := "Client testkit for building http4s clients",
+    startYear := Some(2014),
+    libraryDependencies ++= Seq(
+      munit.value,
+      munitCatsEffect.value,
+    ),
+    mimaPreviousArtifacts := Set.empty,
```

Comment:
No bincompat guarantees.

Reply:
That's what we usually do.  We can revisit if someone is testing a backend outside `org.http4s`, but I've never heard of one.